pkg/descheduler/descheduler_test.go

```diff
@@ -28,6 +34,26 @@ import (
 	"sigs.k8s.io/descheduler/test"
 )
 
+var (
+	podEvictionError     = fmt.Errorf("PodEvictionError")
```

Comment:
This is not necessary to use fmt.Errorf

Reply:
`errors.New("PodEvictionError")` is better :)